.github/files/e2e-tests/create-e2e-projects-matrix.sh

```diff
@@ -30,6 +30,8 @@ if [[ "$GITHUB_EVENT_NAME" == "pull_request" ]]; then
 	done
 elif [[ "$GITHUB_EVENT_NAME" == "push" || "$GITHUB_EVENT_NAME" == "workflow_run" ]]; then
 	PROJECTS_MATRIX=("${PROJECTS[*]}")
+elif [[ "$GITHUB_EVENT_NAME" == "repository_dispatch" ]]; then
+	PROJECTS_MATRIX=('{"project":"Jetpack pre-connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/pre-connection","--retries=2"]}')
```

Comment:
This is only temporary, to test the setup works. In a follow-up it should construct the matrix based on the repository that sent the dispatch and the projects available in `PROJECTS`.